autopilot/autopilot.go

```diff
@@ -68,8 +68,9 @@ type Bus interface {
 type Worker interface {
 	RHPScan(hostKey consensus.PublicKey, hostIP string, timeout time.Duration) (worker.RHPScanResponse, error)
 	RHPForm(renterKey consensus.PrivateKey, hostKey consensus.PublicKey, hostIP string, transactionSet []types.Transaction) (rhpv2.ContractRevision, []types.Transaction, error)
-	RHPRenew(renterKey consensus.PrivateKey, hostKey consensus.PublicKey, hostIP string, contractID types.FileContractID, transactionSet []types.Transaction, finalPayment types.Currency) (rhpv2.ContractRevision, []types.Transaction, error)
+	RHPRenew(fcid types.FileContractID, endHeight uint64, hk consensus.PublicKey, hs rhpv2.HostSettings, renterAddress types.UnlockHash, renterFunds types.Currency, rk consensus.PrivateKey) (rhpv2.ContractRevision, []types.Transaction, error)
```

Comment:
`nit` we should probably try and be super consistent with `renterKey` <-> `rk`, so I would rename all `renterKey` instances to `rk` since we use `hk` for `hostKey` everywhere

Reply:
I can do that next but I definitely don't want to pile it on top of this one.

Reply:
yay, working uploads incoming